branca/element.py

```diff
@@ -71,11 +73,20 @@ def __getstate__(self):
         """
         state: dict = self.__dict__.copy()
         state["_env"] = None
+        state["_template"] = None
         return state
 
     def __setstate__(self, state: dict):
         """Re-add ._env attribute when unpickling"""
         state["_env"] = ENV
+
+        state["_template"] = self._template
```

Comment:
This is a bit of a small optimization. But I'm thinking how to make it clearer what is going on here. How about instead we remove `_template` if it exists. That way we don't have to overwrite it here with `self._template`.

So on line 76: `state.pop('_template', None)`
And remove line 83.

If I'm not mistaken that should be equivalent to the current change, but with one less action.

Reply:
Took me a while to understand, but ok, resolved now !